aviary/interface/test/test_height_energy_mission.py

```diff
@@ -158,32 +160,45 @@ def test_mission_optimize_mach_only(self):
     def test_mission_optimize_altitude_and_mach(self):
         # Test with optimize_altitude flag set to True
         modified_phase_info = self.phase_info.copy()
-        for phase in ["climb_1", "climb_2", "descent_1"]:
+        for phase in ["climb", "cruise", "descent"]:
             modified_phase_info[phase]["user_options"]["optimize_altitude"] = True
             modified_phase_info[phase]["user_options"]["optimize_mach"] = True
+        modified_phase_info['climb']['user_options']['constraints'] = {
+            Dynamic.Mission.THROTTLE: {
+                'lower': 0.2,
+                'upper': 0.9,
```

Comment:
should these limits be easier to set?

Reply:
how so?

Reply:
just noticed this is a test, not the actual height_energy_mission.
I was thinking that the limits should be an input based on the engine, rather than a hard coded value.